client/app/domain/optin/index.js

```diff
@@ -0,0 +1,10 @@
+import component from './optin.component';
+import controller from './optin.controller';
+
+const moduleName = 'domainOptin';
+
+angular.module(moduleName, [])
+  .component('domainOptin', component)
+  .controller('domainOptinCtrl', controller);
```

Comment:
Shouldn't be this instead?
```suggestion
  .controller('DomainOptinCtrl', controller);
```